doc/ref_cert/RC1/chapters/chapter03.md

```diff
@@ -87,27 +87,26 @@ from a Compliance state point. The remaining tests have to pass successfully.
 They cover all together the API testing requirements as asked by
 [RA1 Core OpenStack Services APIs](../../../ref_arch/openstack/chapters/chapter05.md)
 
-According to Jerma, the following software versions are considered here to
+According to Wallaby, the following software versions are considered here to
 verify OpenStack Train selected by CNTT:
 
-| software                | version     |
-|-------------------------|-------------|
-| Functest                | jerma       |
-| Horizon Tempest plugin  | 0.2.0       |
-| Cinder Tempest plugin   | 0.3.0       |
-| Keystone Tempest plugin | 0.3.0       |
-| Heat Tempest plugin     | 1.0.0       |
-| Neutron Tempest plugin  | 0.6.0       |
-| Rally OpenStack         | 1.7.1.dev21 |
-| Tempest                 | 22.0.0      |
+| software                | version |
+|-------------------------|---------|
+| Functest                | wallaby |
```

Comment:
@collivier What about Horizon? 

BTW Lakelse RA1 will also include Cyborg and may be strengthening the OpenStack on Kubernetes and that may result in use of Kubernetes, Ceph and other open-source components.

Reply:
@pgoyal01 well tracked! Horizon tempest plugin was merged in Tempest before Wallby (from a milestone pov as Tempest is released independently
https://github.com/opnfv/functest/commit/f7e12cdd782fdb21a047eb9f009af52d2fa5c8db
https://github.com/openstack-archive/tempest-horizon/commit/6883d8ff5a1cc2c20b4fea7b58ba8fcda12ec035